docs/guides/analyzers/hashR_lookup.md

```diff
@@ -0,0 +1,34 @@
+---
+hide:
+  - footer
+---
+The hashR analyzer is used to lookup sha256 hash values against a hashR database
+that is created using the  [google/hashr](https://github.com/google/hashr) project.
+
+### Use case
+[Hashr](https://github.com/google/hashr) is a tool that extracts files and hashes
+from input sources (e.g. raw disk image, GCE disk image, ISO file, Windows update
+package, .tar.gz file, etc.) and allows you to build your own hash sets based on
+your data sources.
+
+This information can be used during an investigation to flag files that are
+"known-good" files, such as files that are part of the company's golden image.
+This can help to find malicious files faster by removing a lot of noise.
+
+You can read more about a demo use case in the blog post
+["Find the needle faster with hashR data"](https://osdfir.blogspot.com/2022/11/find-needle-faster-with-hashr-data.html)
+
+
+### Configuration
+1. Setup and generate your hashR database following the
+[instructions on GitHub](https://github.com/google/hashr). Additional reading
```

Comment:
I would remove the "additional reading" thing here, either the installation docs in HashR are complete or we need to improve it there